recipes/libpq/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "9.5.3" %}
+
+package:
+    name: libpq
+    version: {{ version }}
+
+build:
+  number: 0
+  skip: True   # [win]
+  # features:
```

Comment:
Yes, that was my hope.  I odn't know how quickly I'll get to it - still fighting postgresql build on win.